app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -103,7 +111,9 @@ final case class WriteStacCatalog(
         val updatedExport =
           exportDefinition.copy(
             exportStatus = ExportStatus.Exported,
-            exportLocation = Some(exportPath)
+            exportLocation = Some(exportPath),
+            expiration =
+              Some(Timestamp.from(Instant.now.plus(Duration.ofDays(30L))))
```

Comment:
Looks like this should be conditional to `isCogExport`, right?

Reply:
😱 yeah definitely oops

Reply:
099bdb8